third_party/java/BUILD

```diff
@@ -227,10 +227,10 @@ java_library(
     name = "apache-pulsar-client",
     srcs = ["Empty.java"],
     exports = [
-        "@maven//:org_apache_pulsar_pulsar_client_shaded",
+        "@maven//:org_apache_pulsar_pulsar_client",
     ],
     deps = [
-        "@maven//:org_apache_pulsar_pulsar_client_shaded",
+        "@maven//:org_apache_pulsar_pulsar_client",
     ],
```

Comment:
Why change the pulsar client in this PR?

Reply:
We might want to put it back to the shaded jar. I think I made this change because of a compilation error. I just checked, and I think we changed the dependency in the WORKSPACE file. Here is the PR: https://github.com/apache/incubator-heron/pull/3536/files

Reply:
Ok I put it back to the shaded version of the jar. Should look more like what it was before the PR we merged 23 days ago.